src/lib/style/components/_user-profile-links.scss

```diff
@@ -17,8 +14,6 @@
   }
 
   .TabContent {
-    min-height: calc(100vh - 100px);
-    max-height: calc(100vh - 100px);;
```

Comment:
Moving container styling to SWC (will stretch container to take 100% of the height of the parent in SWC)